nrf_security/Kconfig.psa

```diff
@@ -78,6 +78,14 @@ config MBEDTLS_USE_PSA_CRYPTO
 	help
 	  Corresponds to MBEDTLS_USE_PSA_CRYPTO setting in mbed TLS config file.
 
+config PSA_ITS_ENCRYPTED
```

Comment:
Curious: Why isn't this a TFM_ option?

Reply:
Mainly because we will support EITS outside of TF-M later. We already have some PRs with the EITS being implemented by the zephyr settings backend. This is intended to be used as a high level Kconfig option which will remove the requirement of TF-M when EITS based on zephyr is merged.